tests/k8s/1-020_validate_notifications/01-install.yaml

```diff
@@ -0,0 +1,56 @@
+apiVersion: argoproj.io/v1alpha1
+kind: ArgoCD
+metadata:
+  name: argocd
+spec:
+  notifications:
+    enabled: true
+---
+apiVersion: v1
+kind: Service
+metadata:
+  name: smtp4dev
+spec:
+  selector:
+    app: smtp4dev
+  ports:
+    - name: smtp
+      protocol: TCP
+      port: 2525
+      targetPort: 2525
+    - name: http
+      protocol: TCP
+      port: 80
+      targetPort: 80
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: smtp4dev
+  labels:
+    app: smtp4dev
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: smtp4dev
+  template:
+    metadata:
+      labels:
+        app: smtp4dev
+    spec:
+      affinity:
+        nodeAffinity:
+          requiredDuringSchedulingIgnoredDuringExecution:
+            nodeSelectorTerms:
+              - matchExpressions:
+                  - key: kubernetes.io/os
+                    operator: In
+                    values:
+                      - linux
+      containers:
+        - name: smtp4dev
+          image: quay.io/jrao/smtplistener:latest
```

Comment:
Maybe better to host the image at https://quay.io/organization/argoprojlabs rather than under a personal account.  Also, the source for smtplistener should probably be hosted somewhere that isn't a personal github account.  Maybe under a new folder in the argocd-operator repo itself?

Reply:
https://github.com/argoproj-labs/argocd-operator/pull/723 <--- PR to add the source under /test/auxiliary

I'll talk to Jann about creating an org under argoprojlabs to host the image itself
Thanks for bringing it up @jopit!

Reply:
moved image to argoprojlabs/argocd-notifications-e2e-smtplistener
Thanks @jopit 
Marking this as resolved